src/data/fullness.txt

```diff
@@ -98,7 +98,7 @@ bowl of eyeballs	2	1	awesome	7-9	0	10-20	0	30 Your Eyes are Peeled! (+25% item d
 bowl of fishysoisse	6	8	awesome	27-29	48-74	67-70	52-70
 bowl of lucky charms	1	1	crappy	1	0	6-9	0	50% chance of leprechaun hatchling
 bowl of maggots	2	1	EPIC	10-12	10-20	0	0
-bowl of mernudo	3	1	good	0	0	0	0	Unspaded 30 Sea Guts
+bowl of mernudo	3	1	good	9	0	0	0
```

Comment:
Isn't it still 30 Sea Guts?

Reply:
no, this went away after Crimbo 2019 ended